bundles/org.openhab.binding.tacmi/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -32,6 +32,10 @@
 		<item-type>Switch</item-type>
 		<label>Digital Input (C.M.I. -> OH)</label>
 		<description>A digital channel sent from C.M.I. to openHAB</description>
+		<tags>
+			<tag>Status</tag>
+			<tag>Opening</tag>
+		</tags>
```

Comment:
I'm not sure about `Opening`. TA is highly configurable, and the meaning of the channel also depends on what is configured on the TA. I would tend to remove any `Semantic Property Tag` and just leave `Semantic Point Tag` for the TA channels.

Some bindings of the TA addon scan the TA configuration and update the channels dynamically. Maybe more tags can be added during this step.
For now I would leave vote for `Status`, `Measurement` here

Reply:
`Status`, `Measurement` is a tautology. But thanks for the feedback about TA channels. I will make some revisions.